src/metabase/xrays/automagic_dashboards/core.clj

```diff
@@ -977,14 +962,12 @@
   ([database] (candidate-tables database nil))
   ([database schema]
    (let [dashboard-templates (dashboard-templates/get-dashboard-templates ["table"])]
-     (->> (apply t2/select [:model/Table :id :schema :display_name :entity_type :db_id]
-                 (cond-> [:db_id (u/the-id database)
-                          :visibility_type nil
-                          :active true]
-                   schema (concat [:schema schema])))
+     (->> (load-tables-with-enhanced-table-stats
+           (cond-> [[:= :db_id (u/the-id database)]
+                    [:= :visibility_type nil]
+                    [:= :active true]]
+             schema (conj [:= :schema schema])))
           (filter mi/can-read?)
```

Comment:
since we are using a `mi/can-read?` i think we need to ensure we pass actual `:model/Table` instances through and not the plain maps that are coming back